src/unstract/sdk/utils/tool_utils.py

```diff
@@ -80,6 +86,21 @@ def load_json(
         loaded_json: dict[str, Any] = json.loads(file_contents)
         return loaded_json
 
+    @staticmethod
+    def dump_json(
```

Comment:
There is already a json_dump in here https://github.com/Zipstack/unstract-sdk/blob/ef00f8149f19c586bd967b8cb0dae6965dc9e1e8/src/unstract/sdk/file_storage/fs_impl.py#L318
Should we merge these?

Reply:
Will taken care later once after the changes added in fs_impl.